base/applications/mspaint/precomp.h

```diff
@@ -53,6 +53,7 @@
 #include "selection.h"
 #include "selectionmodel.h"
 #include "sizebox.h"
+#include "canvas.h"
```

Comment:
What was the motivation to move that include downwards? (Before that it seems likethey were sorted lexicogrpahically).
Don't worry, I wouldn't really mind the moving, I am just curious.

Reply:
Canvas needs sizeboxes.